packages/cinema/data/expressions/en.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
We should run `prettier` in this file, the spaces are not regular, and it isn't really pretty.